src/main/java/Duke.java

```diff
@@ -1,10 +1,112 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Naruto and one day I will be Hokage! \nWhat can i do for you?");
+        Scanner scanner = new Scanner(System.in);
+        Naruto naruto = new Naruto();
+        String lineBreak = "-".repeat(20);
+
+        boolean breakLoop = false;
```

Comment:
Perhaps considering changing the name of this boolean to something that sounds more like a check? e.g. isBreakLoop